### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by samapti63**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <!DOCTYPE html>
 <head>
-    <title>geek</title>
+    <title>samapti</title>
```
Comment (janki63) [inline index.html:3]: give other name for title

**Review by janki63 — COMMENTED** [review]
complete your task

File: index.html (hunk 1 above)
Comment (samapti63) [reply index.html:3]: ok

Comment (janki63) [reply index.html:3]: good

**Pull request closed without merging**